src/run.ts

```diff
@@ -68,9 +68,9 @@ export async function getLatestHelmVersion(): Promise<string> {
             }
          `
       )
-      const releases: string[] = repository.releases.nodes
-         .reverse()
-         .map((node: {tagName: string}) => node.tagName)
+      const releases: string[] = repository.releases.nodes.map(
+         (node: {tagName: string}) => node.tagName
+      )
       const latestValidRelease = releases.find((tag) => isValidVersion(tag))
       if (latestValidRelease) return latestValidRelease
```

Comment:
If we apply the  suggestion on the GQL query, we can now use the attributes to find  what the latest version is. In (untested) code something like this:

```suggestion
      const latestValidRelease: string = repository.releases.nodes
         .find(
            ({tagName, isLatest, isDraft, isPreRelease}) => 
                isValidVersion(tagName) && isLatest && !isDraft && !isPreRelease
          )?.tagName

      if (latestValidRelease) return latestValidRelease
```

(B.t.w. Just  like the  existing  code  this assumes that the _latest_ version is somewhere in the most recent N versions sorted by created_date - which is reasonable, but might not be  a given)

Reply:
Agreed with the last 100 assumption being not ideal. If it can't find it there it will default to the hardcoded one which I think is reasonable enough for now. Ideally we'd traverse over the pages until we do find it but for now this PR seems good enough.